autotest/tools/convert/test_convert.py

```diff
@@ -42,7 +42,7 @@ def convert(config, model_case, cuda_prefix):
             dst_path, '--model-format awq --group-size 128 --trust-remote-code'
         ]),
                                      config,
-                                     model_name,
+                                     model_case,
```

Comment:
mode's mode_case?

Reply:
this is a bugfix, it affect tp args. Fixed in this pr.